### DISCUSSION TITLE
Vecerl + vite + vue.js | index.js error

### DISCUSSION THREAD

**Discussion by timothyshen** [vercel/community#779] [Help] [answered]
Hi all, I am trying to deploy my project on vercel, working fine in local environment, but having issues when deploy. And I could not locate the bug
![image](https://user-images.githubusercontent.com/10960710/184822876-66e555cf-495b-4bc5-a4a9-13f3fe7ceb5e.png)
I am not sure why this error is happening. I have checked and tried several ways, like remove the ```type: modulee``` from package.json.
Or other methods but none of these works. The building process and deployment process went through fine, just the content is not showing [https://creader-dao-test.vercel.app/](https://creader-dao-test.vercel.app/)

**Comment by amyegan** [comment, marked as answer]
Hi @timothyshen! If you share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I will help you identify what is going wrong.

**Reply by timothyshen** [reply]
Hi @amyegan weirdly, I used CLI to deploy the site and it seems to be working. I will mark this as resolved for now. Thanks for replying!

**Reply by cyprianwaclaw** [reply]
What did you enter in CLI, I have the same problem?

https://cyprianwaclaw.vercel.app/ and repo: https://github.com/cyprianwaclaw/Nuxt-Pinia-ts

**Reply by timothyshen** [reply]
I just ran the cmd code on the site and it was working XD
